draft-ietf-rats-corim.md

```diff
@@ -1595,17 +1602,18 @@ Profile (label 6):
 ~~~ cddl
 ECT = {
   ? e: environment-map
-  ? c: claims-map / [ + local-claim ]
+  ? c: [ + local-claim ]
   ? a: [ + $crypto-key-type-choice ]
   ? ns: text
   ? cm: cm-type
   ? p: $profile-type-choice
 }
 local-claim = {
-  le: local-environment
+  ? eid: element-id
   c: claims-map
 }
-local-environment =  bstr / tstr
+element-id =  bstr / tstr / uint / tagged-oid-type / tagged-uuid-type
```

Comment:
the typographical convention for closed type choices is `NAME-type-choice`, so this should look like:

```
element-id-type-choice /= bstr
element-id-type-choice /= tstr
element-id-type-choice /= uint
element-id-type-choice /= tagged-oid-type
element-id-type-choice /= tagged-uuid-type
```

Reply:
Clarifying question: are you suggesting avoiding using the typographical conventions in S3 for CDDL related to internal representation?
If so, I think it'd be better to keep the conventions and use an ad hoc prefix/suffix/infix for internal, e.g.: `__`, like in good old C :-)

Reply:
We agreed to define the internal representation of element ID to match the external representation and to add wording to the effect that element IDs are compared in a byte for byte approach.

Reply:
Fixed in commit: https://github.com/ietf-rats-wg/draft-ietf-rats-corim/commit/0d82b15fffd8cef5d09f4747e5e45bed8dd1caf6